docs/index.md

```diff
@@ -1,11 +1,58 @@
+
 Documentation is an important part of this project, and we take the same open-source approach to the documentation as the code. As such, we welcome community contributions, suggestions and constructive feedback on our documentation. Our documentation is hosted on the [Charmhub forum](https://discourse.charmhub.io/t/github-runner-documentation-overview/7817) to enable easy collaboration. Please use the "Help us improve this documentation" links on each documentation page to either directly change something you see that's wrong, ask a question, or make a suggestion about a potential change via the comments section.
 
 If there's a particular area of documentation that you'd like to see that's missing, please [file a bug](https://github.com/canonical/github-runner-operator/issues).
+
+## Project and community
+
+The GitHub runner charm is a member of the Ubuntu family. It's an open-source project that warmly welcomes community projects, contributions, suggestions, fixes, and constructive feedback.
+
+- [Code of conduct](https://ubuntu.com/community/code-of-conduct)
+- [Get support](https://discourse.charmhub.io/)
+- [Join our online chat](https://chat.charmhub.io/charmhub/channels/charm-dev)
+- [Contribute](Contribute)
+
+Thinking about using the GitHub runner charm for your next project? [Get in touch](https://chat.charmhub.io/charmhub/channels/charm-dev)!
+
+# Navigation
+
+| Level | Path | Navlink |
+| -- | -- | -- |
+| 1 | tutorial | [Tutorial]() |
+| 2 | quick-start | [Quick start](https://discourse.charmhub.io/t/github-runner-docs-quick-start/12441) |
+| 2 | managing-resource-usage | [Managing resource usage](https://discourse.charmhub.io/t/github-runner-docs-managing-resource-usage/12450) |
+| 1 | how-to | [How to]() |
+| 2 | run-on-lxd | [How to run on LXD cloud](https://discourse.charmhub.io/t/github-runner-docs-how-to-run-on-lxd-cloud/12631) |
+| 2 | configure-denylist | [How to restrict self-hosted runner network access]() |
+| 2 | repo-policy | [How to comply with repository policies](https://discourse.charmhub.io/t/github-runner-docs-how-to-comply-with-repository-policies/12440) |
+| 2 | change-token | [How to change GitHub personal access token](https://discourse.charmhub.io/t/github-runner-docs-how-to-change-github-personal-access-token/12451) |
+| 2 | change-path | [How to change repository or organization](https://discourse.charmhub.io/t/github-runner-docs-how-to-change-repository-or-organization/12442) |
+| 2 | contribute | [How to contribute](https://discourse.charmhub.io/t/github-runner-docs-how-to-contribute/7815) |
+| 1 | reference | [Reference]() |
+| 2 | actions | [Actions](https://discourse.charmhub.io/t/github-runner-docs-actions/12443) |
+| 2 | configurations | [Configurations](https://discourse.charmhub.io/t/github-runner-docs-configurations/12444) |
+| 1 | explanation | [Explanation]() |
+| 2 | charm-architecture | [Charm architecture](https://discourse.charmhub.io/t/github-runner-docs-charm-architecture/12446) |
```

Comment:
The navigation table should get auto-generated by `upload-charm-docs`

Reply:
Should the table be deleted in this branch?

Reply:
yeh assuming that we're using `upload-charm-docs` / `discourse-gatekeeper` to publish the docs it will get generated by the action